docs/redirects.md

```diff
@@ -0,0 +1,100 @@
+# Adding redirects in MDN
+
+MDN has a series of redirects, defined in the file
+[`_redirects.txt`](https://github.com/mdn/content/blob/main/files/en-us/_redirects.txt)
+in the `content` repo.
+
+Each line contains a single redirect, in the form:
+
+```bash
+FROM-URL  TO-URL
+```
+
+The relative URL to redirect from, followed by a tab character, followed by the
+relative URL to redirect to. A real example is as follows:
+
+```bash
+/en-US/docs/AJAX  /en-US/docs/Web/Guide/AJAX
+```
+
+Once built, the Yari system then puts all of these redirects in place.
+
+The recommended way to add redirects correctly while avoiding errors is to use
+the `add-redirect` tool available in the [`yari`](https://github.com/mdn/yari)
+and [`content`](https://github.com/mdn/content) repos. Let's see how to use this.
+
+## Preparation
+
+### To run add-redirect from the yari repo
+
+- First of all fork and locally clone the `yari` and `content` repos. If you
+  already have some of them cloned locally, make sure they are up-to-date
+  (e.g. `fetch` the updated content from the origin repos).
+- Install the latest toolset in the `yari` repo using `yarn install`.
+- Create a new branch in the `content` repo, and check
+  it out locally.
+- `cd` into your `yari` repo.
+
+#### Adding the right environment variable to yari
+
+Make sure to add the absolute path to your `content` repos' `files` directory to
+the `CONTENT_ROOT` environment variable in `yari`.
+
+The best way to do this is by setting that value in an `.env` file inside your
+`yari` repo.
+
+This can be done by running the following line inside the root of the yari
+directory:
+
+```bash
+echo 'CONTENT_ROOT=/path/to/content/files' >> .env
+```
+
+Note: If an `.env` file does not already exist, it will be created automatically
+when the above line runs.
+
+### To run add-redirect from the content repo
+
+- First of all fork and locally clone the `content` repo. If you
+  already have it cloned locally, make sure it is up-to-date
+  (e.g. `fetch` the updated content from the origin repos).
+- Install the latest toolset using `yarn install`.
+- Create a new branch in the `content` repo, and check
+  it out locally.
+- `cd` into your `content` repo.
+
+## Adding a redirect
+
+The `add-redirect` tool has the following syntax structure.
+
+Inside `yari`:
+
+```bash
+yarn tool add-redirect <from> <to>
```

Comment:
I don't know if we should encourage this at all. 
It's confusing to potential readers of this guide that there's even a choice. Not to mention that, under the hood, it's actually all the same but named differently (`yarn tool` in yari and `yarn content` in content). 

But then again, this very file itself sits inside Yari. What would be ideal, but perhaps cause more work for you @chrisdavidmills is that this guide should sit inside the mdn/content repo. Perhaps as a branch of the contribution guide.